### PR TITLE
feat: use crud client lib

### PR DESCRIPTION
**Pull request by davidebianchi**

### CODE REVIEW

File: types/rbactypes.go [hunk 1]
```diff
@@ -49,10 +49,6 @@ type Binding struct {
 	Roles             []string  `bson:"roles" json:"roles,omitempty"`
 }
 
-type BindingFilter struct {
```
Comment (Danielecina) [inline types/rbactypes.go:52]: ?

**Review by fredmaggiowski — APPROVED** [review]
LGTM

File: types/rbactypes.go (hunk 1 above)
Comment (fredmaggiowski) [reply types/rbactypes.go:52]: ?

Comment (fredmaggiowski) [reply types/rbactypes.go:52]: (it's no longer used since it has been incapsulated within [the lib](github.com/mia-platform/go-crud-service-client)

**Pull request merged**